eventstore/dynamodb/eventstore.go

```diff
@@ -65,14 +69,26 @@ func (c *EventStoreConfig) provideDefaults() {
 	if c.Region == "" {
 		c.Region = "us-east-1"
 	}
+	if c.Endpoint == "" {
+		c.Endpoint = "http://localhost:8000"
+	}
+	if c.Username == "" {
+		c.Username = "doesnotmatter"
+	}
+	if c.Password == "" {
+		c.Password = "doesnotmatter"
+	}
 }
 
 // NewEventStore creates a new EventStore.
 func NewEventStore(config *EventStoreConfig) (*EventStore, error) {
 	config.provideDefaults()
 
+	creds := credentials.NewStaticCredentials(config.Username, config.Password, "")
```

Comment:
Amazon provides us with replicas of their services that can be run locally without external dependencies. Why not to use them? How will you enable CI for the project?
If I run `go test ./...` in the root folder, then dynamodb package fails. I made it consistent to the approach you have with mongo. As simple as that